source/speechViewer.py

```diff
@@ -96,7 +136,26 @@ def appendText(text):
 	#Otherwise it would be impossible to select text, or even just read it (as a blind person).
 	if _guiFrame.FindFocus()==_guiFrame.textCtrl:
 		return
-	_guiFrame.textCtrl.AppendText(text + "\n")
+
+	#If the mouse is over the Speech Viewer window, don't append the text.  (the text
+	#is still spoken, it just isn't appended into the Speech Viewer window)
+	hwndUnderMouse = win32gui.WindowFromPoint(win32api.GetCursorPos())
+	if hwndUnderMouse == _guiFrame.GetHandle():

```

Comment:
There is a oneliner that fits all cases here:

```suggestion
	if winUser.isDescendantWindow(_guiFrame.GetHandle(), hwndUnderMouse):
```

Reply:
@leonardder done (now using isDescendantWindow())